ROADMAP.md

```diff
@@ -1,7 +1,45 @@
 # KF Serving 2019 Roadmap
-## Q2 2019
+## Q3 2019
+### v0.3 Performance and Stability (ETA: September 1, 2019)
+Objective: "Prevent feature regressions with 80% e2e test coverage against a live Cluster."
+* Automated End-to-End Tests
+    * Execute against a Kubeflow maintained GKE Cluster.
```

Comment:
Should also cover AKS

Reply:
Agreed. Is this something Azure would consider contributing?

Reply:
100% - defer to @rakelkar but we pretty much have to do this